examples/htlc-pyteal-ts/scripts/withdraw/common.ts

```diff
@@ -25,7 +25,7 @@ export async function executeTx(
 	txnParams: wtypes.ExecParams
```

Comment:
You import `executeTx` from `algob/web` but at this files you defined another `executeTx`. The name is same and you got build problem. This is the reason your code can't build. Please fix this problem. @thdailong.